packages/otel/src/exporters/exporter-trace-otlp-proto-fetch.ts

```diff
@@ -1,49 +1,41 @@
 import type { ReadableSpan, SpanExporter } from "@opentelemetry/sdk-trace-base";
-import type { IExportTraceServiceRequest } from "@opentelemetry/otlp-transformer";
-import { createExportTraceServiceRequest } from "@opentelemetry/otlp-transformer/build/src/trace";
+import type { IExportTraceServiceRequest } from "@opentelemetry/otlp-transformer/build/src/trace/internal-types";
+import { createExportTraceServiceRequest } from "@opentelemetry/otlp-transformer/build/src/trace/internal";
 import type { ExportResult } from "@opentelemetry/core";
+import { encodeTraceServiceRequest } from "./proto";
 import { OTLPExporterEdgeBase } from "./otlp-exporter-base";
 import { getDefaultUrl } from "./trace-config";
 import type { OTLPExporterConfig } from "./config";
-import { encodeTraceServiceRequest } from "./proto";
 
-/**
- * OTLP exporter for the `http/protobuf` protocol. Compatible with the "edge" runtime.
- */
 export class OTLPHttpProtoTraceExporter implements SpanExporter {
-  /** @internal */
```

Comment:
FMI, why remove this? I think these are only needed for tsdoc to avoid picking up private things into the API docs.